python/lsst/drp/tasks/reprocess_visit_image.py

```diff
@@ -326,9 +326,22 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
         assert not inputs, "runQuantum got more inputs than expected"
 
         detector_summary = visit_summary.find(detector)
+        msg = ""
         if detector_summary is None:
-            msg = f"Detector {detector} not found in visit summary table; not reprocessing this exposure."
-            raise pipeBase.NoWorkFound(msg)
+            msg += "\n  > No entry for the detector was found in the visit summary table"
+        else:
+            if detector_summary.psf is None:
+                msg += "\n  > the PSF model for the detector is None"
+            if detector_summary.wcs is None:
+                msg += "\n  > the WCS model for the detector is None"
+            if detector_summary.apCorrMap is None:
+                msg += "\n  > the aperture correction model map for the detector is None"
+            if detector_summary.photoCalib is None:
+                msg += "\n  > the photometric calibration model for the detector is None"
```

Comment:
It's not a big deal here, but when composing multi-line strings programmatically, it's usually better to append to a `list` of lines and use `"\n".join(lines)`" to stitch them together.

The main advantage is memory allocation efficiency, which only comes into play when there is a very large number of lines, but I think that makes it good practice for consistency everywhere.

Reply:
Always love a good best-practice coding tip from you 😄